src/main/java/org/apache/commons/collections4/bloomfilter/hasher/function/package-info.java

```diff
@@ -16,7 +16,7 @@
  */
 
 /**
- * Implementations of org.apache.commons.collections4.bloomfilter.hasher.HasherFunction
+ * Implementations of org.apache.commons.collections4.bloomfilter.hasher.function
```

Comment:
This package info is a skeleton. It should really be expanded. For now it could probably be changed to:
```
Provides classes and interfaces to define the shape of a Bloom filter and the conversion
of generic bytes to a hash of bit indexes to be used with a Bloom filter.
```